src/rno-g-acq.c

```diff
@@ -321,8 +321,11 @@ int radiant_configure()
 
   radiant_set_pps_config(radiant,pps_cfg); 
 
-
-
+  uint16_t rf0_clock_delay=round(cfg.radiant.trigger.RF[0].readout_delay*radiant_get_sample_rate(radiant)/(128*1000));
+  uint16_t rf1_clock_delay=round(cfg.radiant.trigger.RF[1].readout_delay*radiant_get_sample_rate(radiant)/(128*1000));
```

Comment:
you probably need to add a range check here?

Reply:
Yes, I didn't add it in the set delay function so I'll add it here. and these need to be uint8_t's... whoops.